frontend/packages/kserve/src/extensions.ts

```diff
@@ -0,0 +1,52 @@
+import type { ModelServingPlatformExtension } from '@odh-dashboard/model-serving/extension-points';
+
+const extensions: ModelServingPlatformExtension[] = [
+  {
+    type: 'model-serving.platform',
+    properties: {
+      id: 'kserve',
+      name: 'KServe',
+      manage: () => import('./managePlatform').then((m) => m.default),
+      enableCardText: {
+        title: 'Single-model serving platform',
+        description:
+          'Each model is deployed on its own model server. Choose this option when you want to deploy a large model such as a large language model (LLM).',
+        selectText: 'Select single-model',
+        enabledText: 'Single-model serving enabled',
+      },
+      deployedModelsView: {
+        startHintTitle: 'Start by deploying a model',
+        startHintDescription: 'Each model is deployed on its own model server',
+        deployButtonText: 'Deploy model',
+      },
+    },
+  },
+  {
+    type: 'model-serving.platform',
+    properties: {
+      id: 'modelmesh-dummy',
+      name: 'ModelMesh (dummy)',
+      manage: () =>
+        import('./managePlatform').then(() => ({
+          isInstalled: () => Promise.resolve(true),
+          isEnabled: () => false,
+          enable: () => Promise.resolve(''),
+          disable: () => Promise.resolve(''),
+        })),
```

Comment:
If you replace this with:
```
      manage: () =>
        new Promise((resolve) => {
          resolve({ foo: true });
        }),
```
There is no typescript error. I think we have a TypeScript problem here... this object is already invalid and I was curious if it was properly typing things... turns out no 😢 

@christianvogt FYI

Reply:
@andrewballantyne i changed the code to what you suggested and I get an error. The fact that the object includes a property named `disable` doesn't make it invalid, the type still satisfies the api contract.
<img width="929" alt="image" src="https://github.com/user-attachments/assets/1d921372-4745-468e-8f96-bbbb3b7a03d5" />

Reply:
Oh -- interesting... must be an issue with my IDE or something.